pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -346,9 +351,9 @@ type ServicePlanSpec struct {
 	// may be supplied binding to an ServiceInstance on this plan.
 	ServiceInstanceCredentialCreateParameterSchema *runtime.RawExtension `json:"serviceInstanceCredentialCreateParameterSchema,omitempty"`
 
-	// ServiceClassRef is a reference to the service class that
+	// ClusterServiceClassRef is a reference to the service class that
 	// owns this plan.
-	ServiceClassRef v1.LocalObjectReference `json:"serviceClassRef"`
+	ClusterServiceClassRef v1.ObjectReference `json:"clusterServiceClassRef"`
```

Comment:
This is becoming an object reference so that we can store the k8s UID and detect mismatches with the reference and the actual object, right?

Reply:
Yes, because the other refs are 'full' references to make sure we know which object it is and not only a name.